resources/migrations/033_add_pulse_channel_enabled_field.yaml

```diff
@@ -0,0 +1,15 @@
+databaseChangeLog:
+  - changeSet:
+      id: 33
```

Comment:
the PR i merged yesterday used id 33, so you'll need to bump this up to 34 and rename the file.